### PR TITLE
Workflow polish

### PR DESCRIPTION
**Pull request by FrederikBornemann**

### CODE REVIEW

File: doc/workflow/workflow.rst [hunk 1]
```diff
@@ -57,7 +57,7 @@ of the main regression equation and the propensity score.
 
 In step 1., we initialize the data-backend and thereby declare the role of the outcome, the treatment and the confounding variables.
 
-We use data from the 1991 Survey of Income and Program Participation which is available via the function fetch_401k [link to docu].
+We use data from the 1991 Survey of Income and Program Participation which is available via the function `fetch_401k <https://docs.doubleml.org/stable/api/generated/doubleml.datasets.fetch_401K.html>`_.
```
Comment (MalteKurz) [inline doc/workflow/workflow.rst:60]: Link both R and Python docu? Additionally the function names are unfortunately not aligned between R and Python.

File: doc/workflow/workflow.rst [hunk 2]
```diff
@@ -167,21 +167,21 @@ We use a regression learner for the outcome variable net financial assets.
...
+1. DML Specifications
 ---------------------
 
 In Step 4., we initialize and parametrize the model object which will later be used to perform the estimation.
 
-We initialize a DoubleMLPLR [link to docu] using the previously generated data-backend. Moreover, we specify the resampling
-(= the number of repetitions and folds for repeated cross-fitting [link to user guide section resampling]),
-the dml algorithm (DML1 vs. DML2 [link to algorithm section in user guide]) and the score function ("partialling out" or
-"IV-type") [link to user guide section on score].
+We initialize a `DoubleMLPLR <https://docs.doubleml.org/stable/api/generated/doubleml.DoubleMLPLR.html>`_ using the previously generated data-backend. Moreover, we specify the resampling
```
Comment (MalteKurz) [inline doc/workflow/workflow.rst:175]: Link to Py and R API docu?

Comment (FrederikBornemann) [reply doc/workflow/workflow.rst]: added in 06afc1d

Comment (PhilippBach) [reply doc/workflow/workflow.rst]: Maybe it's enough to highlight the code as in the user guide and later have a short block with references to the docu? For example as in here https://docs.doubleml.org/stable/guide/data_backend.html and https://docs.doubleml.org/stable/guide/models.html

Comment (PhilippBach) [reply doc/workflow/workflow.rst]: But you can leave this open and I'll fix it once you've integrated it in the dev docu? I think we'll have a final polishing step anyways

**Review by MalteKurz — APPROVED** [review]

File: doc/workflow/workflow.rst (hunk 2 above)
Comment (MalteKurz) [reply doc/workflow/workflow.rst]: Alright, sounds like a good plan.

**Pull request merged**